### ISSUE TITLE
Fix loss tests

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-cisto-ai/tensorflow-caney#22]
Fix loss tests, lookup the actually name of the loss function.
